dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -160,13 +160,17 @@ def test_true_categorical_report(self):
         self.assertTrue(report["categorical"])
         six.assertCountEqual(
             self,
-            ['unique_count', 'unique_ratio', 'categories'], report['statistics']
+            ['unique_count', 'unique_ratio', 'categories', 'categorical_count'], report['statistics']
         )
         self.assertEqual(3, report["statistics"]["unique_count"])
         self.assertEqual(0.25, report["statistics"]["unique_ratio"])
         self.assertCountEqual(
             ["a", "b", "c"], report["statistics"]["categories"]
         )
+        expected_dict = {"a": 3, "b": 4, "c": 5}
+        self.assertDictEqual(expected_dict, report["statistics"]
+        ["categorical_count"])
```

Comment:
instead of moving just the key to a new line:
```python
self.assertDictEqual(expected_dict, 
                 report["statistics"]["categorical_count"])
```